python-sdk/tests/sql/operators/test_append.py

```diff
@@ -182,3 +183,33 @@ def test_append_on_tables_on_different_db(sample_dag, database_table_fixture):
                 source_table=load_append,
             )
         test_utils.run_dag(sample_dag)
+
+
+@pytest.mark.skipif(
+    not DATASET_SUPPORT, reason="Inlets/Outlets will only be added for Airflow >= 2.4"
+)
+def test_inlets_outlets_supported_ds():
+    """Test Datasets are set as inlets and outlets"""
+    input_file = File("gs://bucket/object.csv")
+    output_table = Table("test_name")
```

Comment:
@kaxil is there a way for users to use File/Table in the operator and not produce an outlet? 

Use Case:
If the user wants to load a table and do multiple transformations on the same table and only in the last transformation it is ready to be consumed by another dag. How do we do this right now?

load_file(output_table=Table(name="test")) -> transform_table_1(test:Table) -> transform_table_2(test:Table)

I'm guessing that every transformation step will be producing the same outlet URI.

Reply:
>I'm guessing that every transformation step will be producing the same outlet URI and it will be treated as a different instance of datasets.

If the `uri` is same, Airflow should treat them as the same dataset.